test/azul_test_case.py

```diff
@@ -349,3 +352,15 @@ def setUp(self):
                 with self.assertRaises(KeyboardInterrupt):
                     TC(events).run()
                 self.assertEqual(events, expected_events)
+
+
+class AzulMotoTestCase(TestCase):
```

Comment:
How do we ensure that someone doesn't introduce Moto mocks to their test but forgets to mix this in?